Mathlib/CategoryTheory/SmallObject/Iteration.lean

```diff
@@ -37,19 +38,19 @@ namespace CategoryTheory
 open Category Limits
 
 variable {C : Type*} [Category C] {Φ : C ⥤ C} (ε : 𝟭 C ⟶ Φ)
-  {J : Type u} [LinearOrder J]
+  {J : Type u} [Preorder J]
 
 namespace Functor
 
 namespace Iteration
 
 variable {j : J} (F : { i // i ≤ j } ⥤ C)
```

Comment:
Do people like `Set.Iic j` better?

Reply:
I think this file could make better use of interval notation, but that's outside the scope of this PR.